src/test/java/com/google/cloud/tools/io/FilePermissionsTest.java

```diff
@@ -45,6 +45,16 @@ public void testSubDirectoryCanBeCreated() throws IOException {
   }
   
   @Test
+  public void testSubDirectoryCannotBeCreatedInDevNull()  {
+    try {
+      FilePermissions.verifyDirectoryCreatable(Paths.get("/dev/null/foo/bar"));
```

Comment:
This path is Unix specific. It seems Windows 7 or 8 made `C:\` read-only, but the test will likely succeed if run from a different drive.  Probably just skip the test if on Windows.